docs/aqd.rst

```diff
@@ -0,0 +1,283 @@
+
+\-\-output-json
+  Specifies whether to export the call output in JSON format.
+
+\-\-help, -h
+  This displays a short description of all the options
+
+Subcommands:
+++++++++++++
+
+Upload Subcommand
+-----------------
+
+This subcommand enables the uploading of contracts onto the Polkadot blockchain.
+
+.. code-block:: bash
+
+  aqd polkadot upload --suri //Alice -x flipper.contract --output-json
+
+Instantiate Subcommand
+----------------------
+
+This subcommand facilitates the instantiation of contracts on the Polkadot blockchain.
+
+.. code-block:: bash
+
+  aqd polkadot instantiate --suri //Alice --args true -x --output-json --skip-confirm flipper.contract
+
+Options specific to the ``instantiate`` subcommand:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+\-\-constructor *constructor*
+  Specifies the name of the contract constructor to call. [default: new]
+
+\-\-args *<args>...*
+  Specifies the arguments of the contract constructor to call.
+
+\-\-value *value*
+  Specifies the value to be transferred as part of the call. [default: 0]
+
+\-\-gas *gas*
+  Specifies the maximum amount of gas to be used for this command.
+
+\-\-proof-size *proof-size*
+  Specifies the maximum proof size for this instantiation.
+
+\-\-salt *salt*
+  Specifies a salt used in the address derivation of the new contract.
+
+-y, \-\-skip-confirm
+  When set, skips the interactive confirmation prompt.
+
+Call Subcommand
+---------------
+
+This subcommand enables the calling of contracts on the Polkadot blockchain."
+
+.. code-block:: bash
+
+  aqd polkadot call --contract 5EFYe3hkH2wFK1mLxD5VSqD88hfPZWihXAKeqozZELsL4Ueq --message get --suri //Alice flipper.contract --output-json --skip-confirm
+
+Options specific to the ``call`` subcommand:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+\-\-contract *contract*
+  Specifies the address of the contract to call.
+
+-m, \-\-message *message*
+  Specifies the name of the contract message to call.
+
+\-\-args *<args>...*
+  Specifies the arguments of the contract message to call.
+
+\-\-value *value*
+  Specifies the value to be transferred as part of the call. [default: 0]
+
+\-\-gas *gas*
+  Specifies the maximum amount of gas to be used for this command.
+
+\-\-proof-size *proof-size*
+  Specifies the maximum proof size for this call.
+
+-y, \-\-skip-confirm
+  When set, skips the interactive confirmation prompt.
+
+Remove Subcommand
+-----------------
+
+This subcommand allows for the removal of contracts from the Polkadot blockchain.
+
+.. code-block:: bash
+
+  aqd polkadot remove --suri //Alice --output-json --code-hash 0x94e67200d3d8f0f420873f8d1b426fdf5eb87f208c6e5d061822e017ffaef2a8 flipper.contract
+
+Options specific to the ``remove`` subcommand:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+\-\-code-hash *code_hash*
+  Specifies the code hash to remove.
+
+
+Interacting with Solana running programs on-chain
+_________________________________________________
+
```

Comment:
Same here, this could better explain the concepts of Solana. E.g. what it means to call a contract, how it works with the accounts you have to provide etc.

Reply:
Added a separate paragraph for this